packages/services/src/__tests__/vckit.service.test.ts

```diff
@@ -0,0 +1,252 @@
+import { privateAPI } from '../utils';
+import  { issueCredentialStatus, issueVC }  from '../vckit.service';
+
+describe('issueVC', () => {
+  const mockCredentialStatus = {
+    'id': 'http://example.com/bitstring-status-list/1#0',
+    'type': 'BitstringStatusListEntry',
+    'statusPurpose': 'revocation',
+    'statusListIndex': 0,
+    'statusListCredential': 'http://example.com/bitstring-status-list/1'
+  };
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it('should issue VC successfully without credential status', async () => {
+    jest.spyOn(privateAPI, 'post')
+      .mockResolvedValueOnce({
+        verifiableCredential: {
+          '@context': ['https://www.w3.org/ns/credentials/v2'],
+          type: 'EnvelopedVerifiableCredential',
+          id: 'data:application/vc-ld+jwt,eyJhbGciOiJFZERTQSIsImlzcyI6ImRpZDp3ZWIvcmcvMjAxOC9jcmVkZWyMDIyIn1dfQ.8pUt1rZktWKGBGyJ6GH3io6f7fliAg8IWsEqTWCYvKm0fQkIlPnqqTobxgR3qmtMd_jJXc8IHwbVVOBUEvpcCg',
+        },
+      });
+
+    const vc = await issueVC({
+      context: ['https://www.w3.org/ns/credentials/v2'],
+      type: ['VerifiableCredential', 'Event'],
+      issuer: 'did:example:123',
+      credentialStatus: mockCredentialStatus,
+      credentialSubject: { id: 'did:example:123', name: 'John Doe' },
+      restOfVC: { render: {} },
+      vcKitAPIUrl: 'https://api.vc.example.com',
+    });
+
+    expect(vc).toEqual({
+      '@context': ['https://www.w3.org/ns/credentials/v2'],
+      type: 'EnvelopedVerifiableCredential',
+      id: 'data:application/vc-ld+jwt,eyJhbGciOiJFZERTQSIsImlzcyI6ImRpZDp3ZWIvcmcvMjAxOC9jcmVkZWyMDIyIn1dfQ.8pUt1rZktWKGBGyJ6GH3io6f7fliAg8IWsEqTWCYvKm0fQkIlPnqqTobxgR3qmtMd_jJXc8IHwbVVOBUEvpcCg',
+    });
+  });
+
+  it('should issue VC successfully with credential status', async () => {
+    jest.spyOn(privateAPI, 'post')
+      .mockResolvedValueOnce(mockCredentialStatus)
+      .mockResolvedValueOnce({
+        verifiableCredential: {
+          '@context': ['https://www.w3.org/ns/credentials/v2'],
+          type: 'EnvelopedVerifiableCredential',
+          id: 'data:application/vc-ld+jwt,eyJhbGciOiJFZERTQSIsImlzcyI6ImRpZDp3ZWIvcmcvMjAxOC9jcmVkZWyMDIyIn1dfQ.8pUt1rZktWKGBGyJ6GH3io6f7fliAg8IWsEqTWCYvKm0fQkIlPnqqTobxgR3qmtMd_jJXc8IHwbVVOBUEvpcCg',
+        },
+      });
+
+    const vc = await issueVC({
```

Comment:
Hi, should I add the `credentialStatus` value here? Because I see the test case is `issue VC successfully with credential status` but I can not see any `credentialStatus` field.

Reply:
Yep, I realized the test title doesn’t match the test code. I’ll update the test case title to make it clearer.